qa/integration-tests/src/test/java/io/camunda/zeebe/it/startup/ZeebeUserRootlessImageTest.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.it.startup;
+
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+/** Tests a deployment running with user "zeebe" instead of root. */
+@Testcontainers
+final class ZeebeUserRootlessImageTest extends AbstractRootlessImageTest {
+
+  @Override
+  public String getUser() {
+    return "zeebe";
```

Comment:
:thought_balloon: FYI, putting "1000:0" works. In fact, putting anything as long as you are part of the root group, works.

This is the standard on OpenShift (as described [here](https://cloud.redhat.com/blog/a-guide-to-openshift-and-uids#uids-and-guis-in-action-container-perspective)).

:question: Do you know what's the default for Kubernetes in general? Or do we also want to support this out of the box for plain Docker users?

Reply:
> Or do we also want to support this out of the box for plain Docker users?
exactly, this test is motivated by plain docker security recommendations to make use of an unpriviliged user, e.g. [here](https://cheatsheetseries.owasp.org/cheatsheets/Docker_Security_Cheat_Sheet.html#rule-2-set-a-user), which don't cover group assignment. Thus I deliberately omitted the group assignment.

That's why I wanted to keep it simple by making sure just using the zeebe user is enough to run zeebe, regardless of group assignment. Also if we go for making the zeebe user the default from the dockerfile we wouldn't need to bother about group assignment.

> This is the standard on OpenShift (as described [here](https://cloud.redhat.com/blog/a-guide-to-openshift-and-uids#uids-and-guis-in-action-container-perspective)).
To me the role of the group id `0` is a bit confusing 😅 , e.g. dist docs like the [one from debian](https://wiki.debian.org/SystemGroups) indicate this group is not intended to be used by others but the root user which makes me wonder if the handling by OpenShift is a general one to rely on.
I can also imagine asking customers to assign the root group to the users they run the image with may raise confusion as well.